proto/warden/sched/v1beta1/extension.proto

```
@@ -0,0 +1,6 @@
+syntax = "proto3";
+package warden.sched.v1beta1;
```

Buf (1.54.0)

Files with package "warden.sched.v1beta1" must be within a directory "warden/sched/v1beta1" relative to root but were in directory "proto/warden/sched/v1beta1".

(PACKAGE_DIRECTORY_MATCH)